src/openrct2/ride/TrackDesign.h

```diff
@@ -21,7 +21,7 @@
 struct Ride;
 struct ResultWithMessage;
 
-#define TRACK_PREVIEW_IMAGE_SIZE (370 * 217)
+constexpr uint32_t kTrackPreviewImageSize{ 370 * 217 };
```

Comment:
```suggestion
constexpr uint32_t kTrackPreviewImageSize = 370 * 217;
```
(more consistent with the rest).